packages/cli/test/commands/upgrade.test.ts

```diff
@@ -113,7 +112,7 @@ describe('upgrade:command typescript@next', () => {
     expect(result.stdout).contain(`Rehearsing with typescript@${latestPublishedTSVersion}`);
     expect(existsSync(reportFile)).toBeTruthy();
 
-    const report = readJSONSync(reportFile) as import('@rehearsal/reporter').Report;
+    const report = readJSON(reportFile) as import('@rehearsal/reporter').Report;
```

Comment:
This could be rewritten as:
```suggestion
    const report = readJSON< import('@rehearsal/reporter').Report>(reportFile);
```

Though this is a test and it's minor.